mono/mini/profiler.c

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
It might be better to call this file mini-profiler.c, we already have a profiler.c in metadata/.